File: src/kyu_8/basic_subclasses_adam_and_eve/js/GodTest.js
const Test = require('@codewars/test-compat');
const God = require("./God");
const Man = require("./God");

describe("GodTest", function() {
    it("Adam should be a Man", function() {
        let humans = God.create()
        Test.assertEquals(humans[0] instanceof Man, true, 'Expected Adam to be a Man');
    });
});

File: src/kyu_8/basic_subclasses_adam_and_eve/java/GodTest.java
package kyu_8.basic_subclasses_adam_and_eve.java;

import org.junit.Test;
import static org.junit.Assert.assertEquals;

public class GodTest {
    @Test
    public void makingAdam(){
        Human[] paradise = God.create();
        assertEquals("Adam are a man", true ,paradise[0] instanceof Man);
    }
}
